inspirehep/modules/authors/rest/citations.py

```diff
@@ -46,17 +48,13 @@ def serialize(self, pid, record, links_factory=None):
         author_pid = pid.pid_value
         citations = {}
 
-        search = LiteratureSearch().query({
-            "match": {
-                "authors.recid": author_pid
-            }
-        }).params(
-            _source=[
-                "authors.recid",
-                "control_number",
-                "self",
-            ]
-        )
+        query = Q('match', authors__recid=author_pid)
+        search = LiteratureSearch().query('nested', path='authors', query=query)\
```

Comment:
You edited a number of LiteratureSearch queries. Do we have tests already covering this lines?

Reply:
Yes, these ones: https://github.com/inspirehep/inspire-next/blob/f52318ab40727c836e49aacf23f36985d72f6134/tests/integration/test_author_api.py (which in fact were previously failing: https://travis-ci.org/inspirehep/inspire-next/jobs/357007070#L1471)